core/app/models/workarea/configuration/params.rb

```diff
@@ -17,12 +17,7 @@ def to_h
               value
             end
 
-          memo[field.key] =
-            if formatted_value.blank? && !field.allow_blank?
-              field.default
-            else
-              formatted_value
-            end
+          memo[field.key] = formatted_value.presence || field.default
```

Comment:
Did we ever come to resolution with this from the old PR? Still think this logic is no good in the scenario of field that is not required, with a default value, but an admin wants to it be blank.

Reply:
Oh that must have happened while you were out.

https://github.com/workarea-commerce/workarea/pull/508
https://github.com/workarea-commerce/workarea/pull/512

Logically, there wasn't a way to change the default of `allow_blank` and preserve all the existing functionality, which you felt shouldn't be changed in a patch.